exercises/practice/hamming/.approaches/list-comprehension/snippet.txt

```diff
@@ -0,0 +1,8 @@
+let distance (strand1: string) (strand2: string) : int option =
+    if strand1.Length <> strand2.Length
+    then None
+    else
+        [ for idx in 0 .. strand1.Length - 1 do
+            if strand1[idx] <> strand2[idx] then yield 1 else yield 0 ]
+        |> List.sum
+        |> Some
```

Comment:
Something to keep in mind: lines that are too long are clipped.

<details><summary>some measurements on my end</summary>

- Desktop, overview page: room for 42 columns, clipped to 41 when longer.
- Desktop, individual approach page: room for 36 columns, clipped to 35 when longer.
- Phone, portrait, both pages: room for 30 columns, clipped to 29 when longer.
- Phone, landscape, both pages: plenty of room.

</details>

@ErikSchierboom don't you think it would be better for the snippets to be horizontally scrollable rather than clipped?